src/patterns/molecules/details/details-custom.hbs

```diff
@@ -0,0 +1,45 @@
+---
+name: Custom expand and collapse
+description: The functions driving the details and summary component can be called independantly
+order: 3
+notes: |
+    - Styling must be provided seperately
+    - Details can be initialized using `Mzp.Details.init(selector, options)`, where `options` is an Object with the following properties.
+      - `onDetailsOpen` [Function] callback when a details is opened (optional).
+      - `onDetailsOpen` [Function] callback when a details is closed (optional).
+---
+
+<style>
+.demo-section .is-closed[aria-hidden=true] { display: none; }
+</style>
+
+<section class="demo-section">
+  <h5 class="demo-heading">Heading</h5>
```

Comment:
It should be interactive if the window is 767 or smaller. The idea was to demo custom initialization and destroy. Do you have any ideas for a better demo?

Reply:
Ah I see! This totally escaped me when I was reading the page, and I should have paid closer attention to the code in the example itself. Still, it could perhaps help if the docs described what the example itself is set up to do? I probably wouldn't have been tripped up in assuming something was broken with an accompanying description?